www/base/src/app/common/directives/loginbar/loginbar.tpl.jade

```diff
@@ -15,7 +15,9 @@ ul.nav.navbar-nav.navbar-right(ng-show="config.auth.name != 'NoAuth'")
 
     li.dropdown(uib-dropdown, ng-hide="anonymous")
         a.dropdown-toggle(uib-dropdown-toggle)
-            .avatar(ng-if="email")
+            .avatar(ng-if="username")
```

Comment:
I would pass the two arguments and let the avatar plugin choose what is suitable.

In this case, your implementation would break the GravatarAvatar that buildbot is using.